packages/dashboard/src/dashboard.js

```diff
@@ -48,6 +49,11 @@ function Dashboard({ config, children }) {
     ...externalDesignSystemTheme,
     colors: lightMode,
   };
+  // Set up modal focus trap on app's mount
+
+  document.body.setAttribute('id', 'dashboardWrapper');
+  setAppElement('#dashboardWrapper > div:first-of-type');
```

Comment:
is this equivalent to: `setAppElement('body > div:first-of-type')` ?

Also should this be in an effect so it only runs once?

Reply:
Just want to clarify here, the assumption you're making is that the first div in the body will be the root react element for the dashboard?

Reply:
> should this be in an effect so it only runs once? 

It only runs on initial mount of the `Dashboard` so it's not getting called again as you navigate the dashboard so putting this in an effect doesn't do anything. 

> is this equivalent to: setAppElement('body > div:first-of-type') ?

Yes, but I wanted to add the id so that it reads easier or can be updated more readily if my assumption about the first div in the body being the root element to mount goes sour. Honestly, this should hold true - the way that the Dashboard is getting exported and used will maintain this structure. It feels icky but this was the best way I could come up with grabbing the right element to set for hiding the page when we're in a modal without also hiding the modal _and_ being agnostic of WordPress. I'll play with it a little more today just to triple check.

Reply:
Cool sounds good. I'm not as familiar with how we're actually mounting the dashboard, or how this looks porting to another CMS, so just wanted to check.

Reply:
i'm glad you did, this part feels weird but since it's essentially embedded in WordPress i think it's gonna have to stay a little weird - at least for now. 😿